.github/workflows/deploy-docs.yml

```diff
@@ -57,6 +57,9 @@ jobs:
 
     - run: yarn install --frozen-lockfile
     - run: yarn build
+      env:
```

Comment:
can you confirm whether these two keys will be encrypted on the frontend?

Reply:
It's not encrypted but per Algolia's docs, the search API key is safe for production frontend usage: https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key

<img width="660" height="187" alt="image" src="https://github.com/user-attachments/assets/aedc2b8a-e890-49df-aa0f-143581495ae2" />